Dockerfile

```diff
@@ -0,0 +1,166 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+FROM mcr.microsoft.com/azureml/o16n-base/python-assets:20220331.v12 AS inferencing-assets
+
+# Tag: cuda:11.1.1-devel-ubuntu20.04
+# Env: CUDA_VERSION=11.1.1
+# Env: NCCL_VERSION=2.8.4
+# Env: CUDNN_VERSION=8.0.5.39
+
+FROM nvidia/cuda:11.2.2-cudnn8-devel-ubuntu20.04

```

Comment:
what is this?

I guess you just need to delete 2 files, there are 49 files changed here